infra/charts/turing/values.yaml

```diff
@@ -99,6 +99,11 @@ turing:
   # # Example
   # - name: varlog
   #   mountPath: /var/log
+  clusterConfig:
+    # -- Configuration to use in cluster configuration or authenticate kubernetes with vault configuration
+    useInClusterConfig: false
+    # -- Service account name for turing's kubernetes controller
+    serviceAccountName: turing
```

Comment:
Don't have strong opinion on this, but why do we hardcode it, instead of using `{{ .Release.Namespace }}-svc-acc`? This can be added into `_helpers.tpl` as:
```
{{- define "turing.service-account" -}}
...
{{- end -}}
```

Reply:
sounds good actually, I never knew this existed.